lms/services/exceptions.py

```diff
@@ -32,10 +32,11 @@ class ExternalRequestError(ServiceError):
     :type response: requests.Response
```

Comment:
The docstrings here have gotten out of sync with the actual params.

Reply:
Thanks!